update mjh_hudum only

gregeck opened the pull request:
Initially updated all 5 keyboards.
Then realized that there would be no way for keyman to pull one keyboard at a time.
Reverted all changes to 5 keyboards so as to be consistent with 
  https://github.com/keymanapp/keyboards/tree/master/experimental
Moved m/mjh_hudum to mjh/mjh_hudum
Uploaded changes only specific to mjh_hudum
Will do 4 individual pull requests for aligali, todo, sibe, and manchu after this hudum pull request executed successfully

keyman-server commented:
Thank you for your pull request. You'll see a "build failed" message until the Keyman team has reviewed the pull request and manually initiated the build process.

Every change committed to this branch will become part of this pull request. When you have finished submitting files and are ready for the Keyman team to review this pull request, please post a "Ready for review" comment.

  PLEASE NOTE: team capacity for review is limited in December 2024 and will resume in January 2025. We appreciate your patience.

LornaSIL reviewed (approved):
The keyboard does build. Since we did a move from "m" to "mjh" I'm finding it a bit confusing. 
- Also, there were some "white space" only changes to your archived .kmn files in other projects. Try to only make changes in one keyboard per pull request
- There was a test file you added that needed deleting. I think we had to do that before. You should probably delete it. It was in the "keyboards" folder.
- I changed the version for `1.01` to `1.0.1` as that is the way we prefer versioning to happen. So, for future keyboards use that scheme.
- I moved the recent change in HISTORY.md from the bottom to the top as we prefer to see the latest change to a keyboard at the top (not the bottom)

gregeck commented:
Noted with thanks  on all four mjh_hudum items.
Should I go on to the other four keyboards?

From: Lorna Evans ***@***.***>
Sent: Monday, January 13, 2025 2:13 PM
To: keymanapp/keyboards ***@***.***>
Cc: Greg Eck ***@***.***>; Author ***@***.***>
Subject: Re: [keymanapp/keyboards] update mjh_hudum only (PR #3277)


@LornaSIL approved this pull request.

The keyboard does build. Since we did a move from "m" to "mjh" I'm finding it a bit confusing.

  *   Also, there were some "white space" only changes to your archived .kmn files in other projects. Try to only make changes in one keyboard per pull request
  *   There was a test file you added that needed deleting. I think we had to do that before. You should probably delete it. It was in the "keyboards" folder.
  *   I changed the version for 1.01 to 1.0.1 as that is the way we prefer versioning to happen. So, for future keyboards use that scheme.
  *   I moved the recent change in HISTORY.md from the bottom to the top as we prefer to see the latest change to a keyboard at the top (not the bottom)

—
Reply to this email directly, view it on GitHub<https://github.com/keymanapp/keyboards/pull/3277#pullrequestreview-2548052379>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AOBLFBHI5KWBAOQ44GPBZK32KQ3ARAVCNFSM6AAAAABVDM35I6VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDKNBYGA2TEMZXHE>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>

LornaSIL commented:
Try another one.

On Mon, Jan 13, 2025, 5:38 PM Greg Eck ***@***.***> wrote:

> Noted with thanks on all four mjh_hudum items.
> Should I go on to the other four keyboards?
>
> From: Lorna Evans ***@***.***>
> Sent: Monday, January 13, 2025 2:13 PM
> To: keymanapp/keyboards ***@***.***>
> Cc: Greg Eck ***@***.***>; Author ***@***.***>
> Subject: Re: [keymanapp/keyboards] update mjh_hudum only (PR #3277)
>
>
> @LornaSIL approved this pull request.
>
> The keyboard does build. Since we did a move from "m" to "mjh" I'm finding
> it a bit confusing.
>
> * Also, there were some "white space" only changes to your archived .kmn
> files in other projects. Try to only make changes in one keyboard per pull
> request
> * There was a test file you added that needed deleting. I think we had to
> do that before. You should probably delete it. It was in the "keyboards"
> folder.
> * I changed the version for 1.01 to 1.0.1 as that is the way we prefer
> versioning to happen. So, for future keyboards use that scheme.
> * I moved the recent change in HISTORY.md from the bottom to the top as we
> prefer to see the latest change to a keyboard at the top (not the bottom)
>
> —
> Reply to this email directly, view it on GitHub<
> https://github.com/keymanapp/keyboards/pull/3277#pullrequestreview-2548052379>,
> or unsubscribe<
> https://github.com/notifications/unsubscribe-auth/AOBLFBHI5KWBAOQ44GPBZK32KQ3ARAVCNFSM6AAAAABVDM35I6VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDKNBYGA2TEMZXHE>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.******@***.***>>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/keymanapp/keyboards/pull/3277#issuecomment-2588463672>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABV6RI2ZHGSZGPNDF2ZTA5D2KRE6LAVCNFSM6AAAAABVDM35I6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKOBYGQ3DGNRXGI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

mcdurdin commented:
> * I changed the version for `1.01` to `1.0.1` as that is the way we prefer versioning to happen. So, for future keyboards use that scheme.

Please note -- this is not just a preference. `1.01` is treated the same as `1.1` -- prefix zeroes are ignored. See https://semver.org for more detail.
